docs/source/metadata.rst

```diff
@@ -75,13 +79,14 @@ Position Metadata
 -----------------
 Position (line and column numbers) metadata are accessible through the metadata
 interface by declaring the one of the following providers as a dependency. For
-most cases, :class:`~libcst.SyntacticPositionProvider` is what you probably want.
-Accessing position metadata through the :class:`~libcst.MetadataDepedent`
-interface will return a :class:`~libcst.CodeRange` object. See
+most cases, :class:`~libcst.metadata.PositionProvider` is what you probably
+want.
+
+Node positions are is represented with :class:`~libcst.CodeRange` objects. See
 :ref:`the above example<libcst-metadata-position-example>`.
 
-.. autoclass:: libcst.metadata.BasicPositionProvider
-.. autoclass:: libcst.metadata.SyntacticPositionProvider
+.. autoclass:: libcst.metadata.WhitespaceInclusivePositionProvider
+.. autoclass:: libcst.metadata.PositionProvider
```

Comment:
Same comment here. Felt a bit weird to me that we want `PositionProvider` to be the default, but it comes second on the list.